base/replicator_test.go

```diff
@@ -18,6 +19,7 @@ func TestReplicator(t *testing.T) {
 	}
 	_, err := r.Replicate(params, false)
 	goassert.Equals(t, err, nil)
+	time.Sleep(time.Millisecond * 100)
```

Comment:
Switch this to a 20x loop that sleeps for 100 ms each time and rechecks activeTasks, to ensure we don't hit failures under race.